### PR TITLE
feat: footer

### LINKED ISSUE #10: Create Footer
Based on Figma designs.

### PR DESCRIPTION
**Pull request by err53**
Closes #10

~~Waiting on profile data from DeltaHacks members~~ will be added in a forthcoming PR

## Requirements
- [x] Dinos Image
- [x] Team Member Marquee
    - [x] Team Member Quotes
- [x] Social Links on Mobile
    - [x] Make the social links actually clickable
    - [x] Update social link icon based on revisions from Design team
- [x] Footer Links

### CODE REVIEW

File: src/sections/Footer.tsx [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
+      <Image src={snowman} alt="Snowman footer image" />
+      <p className="pb-4 text-center text-2xl">
+        Made with ♥ from the DeltaHacks Team{" "}
+      </p>
+      <Slider />
+      <div className="mt-32 flex w-full max-w-screen-lg flex-col gap-3 py-12 text-center text-xl lg:flex-row lg:gap-8 lg:border-t lg:border-black">
+        <Link
+          href="https://github.com/DeltaHacks/deltahacks-website-2024"
+          className="hover:underline"
+        >
```
Comment (Krish120003) [inline src/sections/Footer.tsx]: This is a flexbox, this should be using gap for spacing instead of margins from space-x-8.

Comment (err53) [reply src/sections/Footer.tsx]: I fixed it in the commit directly before your review :)

**Comment by Krish120003** [comment]
Opted to hide the Team section until we have the images from everyone

**Comment by kabirsg** [comment]
> Opted to hide the Team section until we have the images from everyone

We should have everyone's picture by Sunday EOD. Ill send a reminder out at some point on Sunday too but hopefully we can get it by then

**Comment by kabirsg** [comment]
This looks good to me. Btw the privacy policy and code of conduct are good and there's nothing else to add. No travel policy or anything like that.

File: next.config.js [hunk 2]
```diff
@@ -30,6 +30,10 @@ const config = {
       },
     ];
   },
+  images: {
+    dangerouslyAllowSVG: true,
```
Comment (Krish120003) [inline next.config.js]: Why do we need this?

**Review by kabirsg — APPROVED** [review]
Looks good to me

**Pull request merged**